test/issue427/Makefile

```diff
@@ -1,4 +1,7 @@
-regenerate:
+test:
```

Comment:
Can we move this to only be under the `testall` target.
I don't want the `test` target to download anything.

And could we move the protoc-min-version command into a Makefile in test/issue247?

Reply:
my bad, wrong diff